src/lib/util/path.ts

```diff
@@ -0,0 +1,13 @@
+import * as nodePath from 'path';
+
+// by exporting this way, we can add custom functions and still be type safe
+export const path = {
+  ...nodePath,
+  ensureUnixPath: (path?: string): string | null => {
+    if (!path) {
+      return null;
+    }
+
+    return path.replace(/\\/g, '/');
```

Comment:
Hmpf. Ok. Isn't there a node API for reading or normalizing a platform-independent path?

If not, we can stick with the regex replace but I'd like to use [`path.sep`](https://nodejs.org/api/path.html#path_path_sep) then.

Wdyt?

Reply:
I tried the `path.posix.normalize` api and found that it seems to resolve things like `/a/../b` to be `/b`, but it didn't seem to make path separators uniform. Maybe I was using it wrong. Node isn't my main platform. I agree with the `path.sep` change if we keep the `.replace`.